rest/access-tokens/sync-example/sync-example.5.x.rb

```diff
@@ -0,0 +1,31 @@
+require 'twilio-ruby'
+
+# Required for any Twilio Access Token
+# To set up environmental variables, see http://twil.io/secure
+account_sid = ENV['TWILIO_ACCOUNT_SID']
+api_key = ENV['TWILIO_API_KEY']
+api_secret = ENV['TWILIO_API_KEY_SECRET']
+twilioSyncService = ENV['TWILIO_SYNC_SERVICE_SID']
```

Comment:
we are using for the variables also snake_case convention, let's use it also for `twilio_sync_service`.